components/emoji_picker/emoji_picker_tabs.tsx

```diff
@@ -11,29 +10,43 @@ import {makeAsyncComponent} from 'components/async_load';
 import EmojiPicker from 'components/emoji_picker';
 import EmojiPickerHeader from 'components/emoji_picker/components/emoji_picker_header';
 
+import {Emoji} from '@mattermost/types/emojis';
+
 const GifPicker = makeAsyncComponent('GifPicker', React.lazy(() => import('components/gif_picker/gif_picker')));
 
-export default class EmojiPickerTabs extends PureComponent {
-    static propTypes = {
-        style: PropTypes.object,
-        rightOffset: PropTypes.number,
-        topOffset: PropTypes.number,
-        leftOffset: PropTypes.number,
-        placement: PropTypes.oneOf(['top', 'bottom', 'left', 'right']),
-        customEmojis: PropTypes.object,
-        onEmojiClose: PropTypes.func.isRequired,
-        onEmojiClick: PropTypes.func.isRequired,
-        onGifClick: PropTypes.func,
-        enableGifPicker: PropTypes.bool,
-    };
+type PickerStyle = {
+    top: number;
+    bottom: number;
+    left?: number;
+    right?: number;
+}
+
+export interface Props {
+    style?: PickerStyle;
```

Comment:
this style is actually `React.CSSProperties` if you see 
https://github.com/mattermost/mattermost-webapp/pull/11401/files#diff-fc703a1770091015a25d4016640ec6e6a3531ecda84d59a915f9af5f55aaed9cL76
its being passed down to `Tabs` as style object